mycroft/skills/core.py

```diff
@@ -401,6 +401,9 @@ def shutdown(self):
 
 class FallbackSkill(MycroftSkill):
     fallback_handlers = {}
+    folders = {}
+    override = skills_config.get("fallback_override", False)
+    order = skills_config.get("fallback_priority", [])
```

Comment:
i like the override parameter, i may want to have the override list there but only toggle it sometimes, with the PR for changing config at runtime from skills this can even be changed vocally